src/CredentialModel.cpp

```diff
@@ -244,9 +226,34 @@ void CredentialModel::load(const QJsonArray &json)
             // Update login item address
             pLoginItem->setAddress(bAddress);
 
-            // Update login favorite
-            int iFavorite = cnode["favorite"].toInt();
-            pLoginItem->setFavorite(iFavorite);
+            if (isFido)
+            {
+                pLoginItem->setCategory(0);
```

Comment:
what's that for? :)

Reply:
Without that category value is not initialized and it had have a value >0 so the category icon was displayed.